eng/generate-compiler-code.cs

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env dotnet
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Need this fix to delete the static graph disable: https://github.com/dotnet/sdk/pull/50532
```

Comment:
Should we link to a follow-up issue rather than a merged PR?

Reply:
I'd feel better with an issue so we don't forget. If you're confident you'll remember to address this in next couple months, than this is fine

Reply:
I think it'll be fine, I expect to use `dotnet run file` in the repo again :).